caom2pipe/caom_composable.py

```diff
@@ -237,7 +237,9 @@ def build_chunk_energy_range(chunk, filter_name, filter_md):
     cw = ac.FilterMetadataCache.get_central_wavelength(filter_md)
     fwhm = ac.FilterMetadataCache.get_fwhm(filter_md)
     if cw is not None and fwhm is not None:
-        resolving_power = ac.FilterMetadataCache.get_resolving_power(filter_md)
+        resolving_power = ac.FilterMetadataCache.get_resolving_power(
```

Comment:
I think we should go 120 wide rows. What do you think?

Reply:
120 rows it is.